src/components/CallHistoryLastUpdatedStatus.tsx

```diff
@@ -18,14 +18,24 @@ const CallHistoryLastUpdatedStatus = (): ReactElement => {
         });
     }, []);
 
+    const DisplayResult = () => {
+        const date = mirBstDateFormatter(reportStatus, true);
```

Comment:
On naming:
1. Is the `mir` necessary? It's somewhat implied in this repo 😁 
2. Is the `BST` necessary? Maybe the format will change one day.
3. "date formatter" sounds like a thing (e.g. object) rather than an action (i.e. function). I'd suggest one of the following:
    - Describe the action it performs - e.g.`formatDate(reportStatus)`
    - Describe the thing it returns - e.g. `formattedDate(reportStatus)`
    - If using "date formatter" it would be a class with method on it - e.g. `dateFormatter.format(reportStatus)`